script/test/cmd/tests.sh

```diff
@@ -235,6 +239,6 @@ convert::expect_success "kompose --provider=openshift convert --stdout -j" "$KOM
 cd $CURRENT_DIR
 
 # Removes generated output
-rm -rf /tmp/output-os.json
+rm -rf /tmp/output*.json
```

Comment:
recursive and force plus a `*` = bad idea. very dangerous.
instead, please add:
```
rm /tmp/output-os.json
rm /tmp/output-buildarg-os.json
```